### PR TITLE
Added missing smoke bit in replay frame keys enum

### PR DESCRIPTION
**Pull request by Syriiin**
### Description

Add missing smoke key bit and fixed mod list to replace NoVideo with TouchDevice in osr file format documentation (EN, FR, ZH)

### Status

Finished

### Changelog

- Add missing smoke key bit in .osr file format documentation (EN, FR, ZH)
- Changed NoVideo mod to TouchDevice in mod list in .osr file format documentation (EN, FR, ZH)

### CODE REVIEW

File: wiki/osu!_File_Formats/Osr_(file_format)/zh.md [hunk 1]
```diff
@@ -51,7 +51,7 @@
 | w    | Long     | 距离上个动作的时间（毫秒）                             |
 | x    | Float    | 鼠标的X坐标（从0到512）                                |
 | y    | Float    | 鼠标的Y坐标（从0到384）                                |
-| z    | Integer  | 鼠标、键盘按键的组合 (M1 = 1, M2 = 2, K1 = 4, K2 = 8) (K1总是与M1一起，K2总是与M2一起。1+4=5 2+8=10。) |
+| z    | Integer  | 鼠标、键盘按键的组合 (M1 = 1, M2 = 2, K1 = 4, K2 = 8, 烟雾 = 16) (K1总是与M1一起，K2总是与M2一起。1+4=5 2+8=10。) |
```
Comment (kj415j45) [inline wiki/osu!_File_Formats/Osr_(file_format)/zh.md]: smoke 目前没有合适的翻译。

the 'smoke' 's translation haven't been confirm.


需要 @osu-translate-zh/translator 帮助确认。

Comment (Deardrops) [reply wiki/osu!_File_Formats/Osr_(file_format)/zh.md]: smoke translate to `烟雾轨迹` in [help center](https://github.com/ppy/osu-wiki/blob/6b9d29286f709fb78feb8ac0ae06cdfc2315c25b/wiki/Help_Center/zh.md#我的光标后的白色尾巴烟雾轨迹是什么).
but in this line, it's better to use the original.

Comment (Syriiin) [reply wiki/osu!_File_Formats/Osr_(file_format)/zh.md]: I asked a chinese player what would work in this context and this is what I was told, I cant confirm it myself.

Comment (kj415j45) [reply wiki/osu!_File_Formats/Osr_(file_format)/zh.md]: @Syriiin , i'll ask more members of our group to confirm it. when we get done, i'll post the discussion result here.

**Comment by Syriiin** [comment]
Rebased and updated to NoVideo to TouchDevice in mod list (EN, FR, ZH)

**Pull request merged**